package.json

```diff
@@ -0,0 +1,133 @@
+    "yamljs": "^0.3.0"
+  },
+  "name": "leaderboard",
+  "description": "",
+  "version": "1.0.0",
+  "private": true,
+  "dependencies": {
+    "acorn": "^8.5.0",
+    "acorn-import-assertions": "^1.7.6",
+    "ajv": "^6.12.6",
+    "ajv-keywords": "^3.5.2",
+    "browserslist": "^4.17.0",
+    "buffer-from": "^1.1.2",
+    "caniuse-lite": "^1.0.30001255",
+    "chrome-trace-event": "^1.0.3",
+    "clone-deep": "^4.0.1",
+    "colorette": "^1.4.0",
+    "commander": "^2.20.3",
+    "cross-spawn": "^7.0.3",
+    "electron-to-chromium": "^1.3.834",
+    "enhanced-resolve": "^5.8.2",
+    "envinfo": "^7.8.1",
+    "es-module-lexer": "^0.7.1",
+    "escalade": "^3.1.1",
+    "eslint-scope": "^5.1.1",
+    "esrecurse": "^4.3.0",
+    "estraverse": "^4.3.0",
+    "events": "^3.3.0",
+    "execa": "^5.1.1",
+    "fast-deep-equal": "^3.1.3",
+    "fast-json-stable-stringify": "^2.1.0",
+    "fastest-levenshtein": "^1.0.12",
+    "find-up": "^4.1.0",
+    "function-bind": "^1.1.1",
+    "get-stream": "^6.0.1",
+    "glob-to-regexp": "^0.4.1",
+    "graceful-fs": "^4.2.8",
+    "has": "^1.0.3",
+    "has-flag": "^4.0.0",
+    "human-signals": "^2.1.0",
+    "import-local": "^3.0.2",
+    "interpret": "^2.2.0",
+    "is-core-module": "^2.6.0",
+    "is-plain-object": "^2.0.4",
+    "is-stream": "^2.0.1",
+    "isexe": "^2.0.0",
+    "isobject": "^3.0.1",
+    "jest-worker": "^27.1.1",
+    "json-parse-better-errors": "^1.0.2",
+    "json-schema-traverse": "^0.4.1",
+    "kind-of": "^6.0.3",
+    "loader-runner": "^4.2.0",
+    "locate-path": "^5.0.0",
+    "lodash": "^4.17.21",
+    "merge-stream": "^2.0.0",
+    "mime-db": "^1.49.0",
+    "mime-types": "^2.1.32",
+    "mimic-fn": "^2.1.0",
+    "neo-async": "^2.6.2",
+    "node-releases": "^1.1.75",
+    "npm-run-path": "^4.0.1",
+    "onetime": "^5.1.2",
+    "p-limit": "^3.1.0",
+    "p-locate": "^4.1.0",
+    "p-try": "^2.2.0",
+    "path-exists": "^4.0.0",
+    "path-key": "^3.1.1",
+    "path-parse": "^1.0.7",
+    "pkg-dir": "^4.2.0",
+    "punycode": "^2.1.1",
+    "randombytes": "^2.1.0",
+    "rechoir": "^0.7.1",
+    "resolve": "^1.20.0",
+    "resolve-cwd": "^3.0.0",
+    "resolve-from": "^5.0.0",
+    "safe-buffer": "^5.2.1",
+    "schema-utils": "^3.1.1",
+    "serialize-javascript": "^6.0.0",
+    "shallow-clone": "^3.0.1",
+    "shebang-command": "^2.0.0",
+    "shebang-regex": "^3.0.0",
+    "signal-exit": "^3.0.3",
+    "source-map": "^0.6.1",
+    "source-map-support": "^0.5.20",
+    "strip-final-newline": "^2.0.0",
+    "supports-color": "^8.1.1",
+    "tapable": "^2.2.0",
+    "terser": "^5.7.2",
+    "terser-webpack-plugin": "^5.2.4",
+    "uri-js": "^4.4.1",
+    "v8-compile-cache": "^2.3.0",
+    "watchpack": "^2.2.0",
+    "webpack-merge": "^5.8.0",
+    "webpack-sources": "^3.2.0",
+    "which": "^2.0.2",
+    "wildcard": "^2.0.0",
+    "yocto-queue": "^0.1.0"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```

Comment:
- [ ] npm run build throws an error. See the error below. Please fix this.
```
$ npm run build

> leaderboard@1.0.0 build
> webpack

[webpack-cli] Failed to load 'C:\Users\PC\Desktop\TSE\Leaderboard\webpack.config.js' config
[webpack-cli] Error: Cannot find module 'html-webpack-plugin'

```
- [ ] Also your npm should run scripts like the following. It does not need to be exact as long as there is a way to start the project say `npm run start`
```
  "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
```